src/types/tab.ts

```diff
@@ -0,0 +1,5 @@
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
```

Comment:
tip: you can use **react-app-env.d.ts** file for types and in this case you don't need export/import